Android.mk

```diff
@@ -60,6 +60,11 @@ endif
 endif
 endif
 
+ifeq ($(strip $(BOARD_USE_MOTO_SF)), true)
+USE_MOTO_SF = 1
+else
+USE_MOTO_SF = 0
```

Comment:
Random observation: looks like this USE_MOTO_SF variable is not used for anything

Reply:
it's used in here

https://github.com/sailfishos/droidmedia/pull/128/files#diff-9a2197d86880c407035450c08008c0fae1973973cb67b26ff0e9f3f854b34f5dR40

Reply:
Oh wait i see what you mean, yeah you're right

Reply:
Also there's no need for this check. Please check the MTK hack that I added for Audiopolicy.

You can do:
```
ifeq ($(FOO),1)
LOCAL_CPP_FLAGS=-DFOO
endif
```

Please change.